src/molecules/Site.h

```diff
@@ -124,9 +125,11 @@ class LJcenter : public Site {
 	 */
 	void readXML(XMLfileUnits& xmlconfig) {
 		Site::readXML(xmlconfig);
+		Log::global_log->info() << "Site type: Lennard-Jones 12-6" << std::endl;
```

Comment:
The output of the type is now done twice: [Here](https://github.com/ls1mardyn/ls1-mardyn/blob/801675ab0786b0cfad312cbbcd1ff30d04c34f5b/src/molecules/Component.cpp#L48) and in the Site class.
This is also the case for the other site types.

Reply:
I think it is good to have the human readable site type in the output just right before the specific parameters.
The first one is in Components and related to the switch therein ... so I would leave it as is.